deployment/kubernetes/helm/pulsar/templates/bookkeeper-configmap.yaml

```diff
@@ -32,7 +32,7 @@ metadata:
 data:
   zkServers:
     {{- $global := . }}
-    {{ range $i, $e := until (.Values.zookeeper.replicaCount | int) }}{{ if ne $i 0 }},{{ end }}{{ template "pulsar.fullname" $global }}-{{ $global.Values.zookeeper.component }}-{{ printf "%d" $i }}.{{ template "pulsar.fullname" $global }}-{{ $global.Values.zookeeper.component }}{{ end }}
+    {{ range $i, $e := until (.Values.zookeeper.replicaCount | int) }}{{ if ne $i 0 }},{{ end }}{{ template "pulsar.fullname" $global }}-{{ $global.Values.zookeeper.component }}-{{ printf "%d" $i }},{{ template "pulsar.fullname" $global }}-{{ $global.Values.zookeeper.component }}{{ end }}
```

Comment:
I don't think this change is correct.

Before this change, it generates `zkServers` as the following settings:
```
pulsar-dev-zookeeper-0.pulsar-dev-zookeeper,pulsar-dev-zookeeper-1.pulsar-dev-zookeeper,pulsar-dev-zookeeper-2.pulsar-dev-zookeeper
```

But after your change, it generates a wrong list:

```
pulsar-dev-zookeeper-0,pulsar-dev-zookeeper,pulsar-dev-zookeeper-1,pulsar-dev-zookeeper,pulsar-dev-zookeeper-2,pulsar-dev-zookeeper
```

Although the wrong list can be used without problems, it doesn't seem to the right fix.